tempto-core/src/main/java/io/prestosql/tempto/hadoop/hdfs/HdfsClient.java

```diff
@@ -21,6 +21,7 @@
 import java.io.InputStream;
```

Comment:
Maybe you could add some example test that test this .

Reply:
Oh - do we have product tests for tempto?
We have tempto.examples. But are thy run actually?

Reply:
Yes, they are run. See the travis: https://travis-ci.com/github/prestosql/tempto/builds/17961737

Reply:
Added.